migrations/20230201519590_add_oauth_state_table.up.sql

```diff
@@ -0,0 +1,12 @@
+-- auth.oauth_state definition
+create table if not exists {{ index .Options "Namespace" }}.oauth_state(
+       id uuid primary key,
+       internal_auth_code varchar(255) null,
```

Comment:
I feel like this `internal_auth_code` could have a better name. At a first glance, I'm not sure to whom this is internal to?

Reply:
We could just call it auth code as well I guess. Was hoping to distinguish it from the `external` auth codes from the providers.

Open to better names or perhaps we can also call it auth code and leave a comment that it refers to an auth code issued by GoTrue